msg/IntersubCom.msg

```diff
@@ -1,5 +1,7 @@
-bool kill_state
-bool mission_State
-uint8 depth
+bool kill_switch_state
+bool mission_switch_state
+uint16 depth
 uint8 mission_id
-uint8 mission_state
\ No newline at end of file
+int8 mission_state
+uint8 torpedos_state
+uint8 droppers_state
```

Comment:
Maybe we could add a bool missions_complete_state ?

Reply:
Shouldn't be a problem. I will create an issue